### PR TITLE
more f429 business

### PR DESCRIPTION
**Pull request by mck1117**
- missed a comma in firmware yaml
- Add vscode build config
- LED pins
- Init ethernet pins at the correct time
- Minor tweaks that let you build without LTO (easier debugging maybe?)

Ethernet tested working on default build for both F429 and H743 nucleo boards.

### CODE REVIEW

File: firmware/console/ethernet_console.cpp [hunk 1]
```diff
@@ -81,23 +81,6 @@ struct EthernetThread : public TunerstudioThread {
 static EthernetThread ethernetConsole;
 
 void startEthernetConsole() {
-#ifndef STM32H7
-	// TODO: why does this break H7? I thought the pins were the same?
```
Comment (rusefillc) [inline firmware/console/ethernet_console.cpp:85]: Does this remove init from nucleo nucleo-F7 config?

**Review by rusefillc — CHANGES_REQUESTED** [review]

File: firmware/console/ethernet_console.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/console/ethernet_console.cpp:85]: In fact, now they just use the same file.

**Comment by mck1117** [comment]
current status:

image

File: firmware/console/ethernet_console.cpp [hunk 2]
```diff
@@ -81,23 +81,6 @@ struct EthernetThread : public TunerstudioThread {
...
-	efiSetPadMode("ETH_RMII_REF_CLK",  Gpio::A1, PAL_MODE_ALTERNATE(0xb));
```
Comment (rusefillc) [inline firmware/console/ethernet_console.cpp:86]: you've killed my fancy pin names

not that those are really needed

Comment (mck1117) [reply firmware/console/ethernet_console.cpp:86]: those fancy names use extra flash memory 🙂

**Pull request merged**